pyk/src/pyk/kcfg.py

```
@@ -537,6 +589,10 @@ def add_verified(self, source_id: str, target_id: str) -> None:
         target_id = self._resolve(target_id)
         self._verified.add((source_id, target_id))
 
+    def add_alias(self, name: str, id: str) -> None:
+        id = self._resolve(id)
+        self._aliases[name] = id
```

```suggestion
    def add_alias(self, node_id: str, alias: str) -> None:
        node_id = self._resolve(node_id)
        self._aliases[alias] = node_id
```

---

This method should probably raise an exception if the alias is already present.